src/main/java/players/mcts/MCTSParams.java

```diff
@@ -1,19 +1,23 @@
 package players.mcts;
 
-import core.AbstractParameters;
+import core.*;
 import core.interfaces.ITunableParameters;
+import org.json.simple.*;
+import org.json.simple.parser.JSONParser;
+import players.PlayerConstants;
 import players.PlayerParameters;
+import players.simple.RandomPlayer;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
+import java.io.FileReader;
+import java.util.*;
 
 public class MCTSParams extends PlayerParameters implements ITunableParameters {
 
     public double K = Math.sqrt(2);
     public int rolloutLength = 10;
     public boolean rolloutsEnabled = false;
     public double epsilon = 1e-6;
+    public String rolloutType = "Random";
```

Comment:
I think this should be a constant and not a string, to avoid misspelling.

Reply:
Yup. done.

Reply:
At the moment we only have one possible value, and no way to change ths default :-)

On that basis, can we leave this to later?

Reply:
But it's ugly!

Reply:
All done in the next commit...